test/e2e/e2e_suite_test.go

```diff
@@ -114,7 +113,7 @@ func init() {
 
 func TestE2E(t *testing.T) {
 	RegisterFailHandler(Fail)
-	junitPath := filepath.Join(artifactFolder, fmt.Sprintf("junit.e2e_suite.%d.xml", config.GinkgoConfig.ParallelNode))
+	junitPath := filepath.Join(artifactFolder, fmt.Sprintf("junit.e2e_suite.%d.xml", GinkgoParallelProcess()))
 	junitReporter := reporters.NewJUnitReporter(junitPath)
 	RunSpecsWithDefaultAndCustomReporters(t, "capi-operator-e2e", []Reporter{junitReporter})
```

Comment:
I thought custom reporters were a thing of the past? I think ginkgo handles the parallel node thing for us now

Reply:
Yeah, you are right, I've fixed this part